decentraland/unity-renderer
# fix: Non-supported characters are displayed incorrectly for notifications

## What does this PR change?
The squares indicating the non-supported characters were not replaced with question marks in the notifications.

## How to test the changes?
1. Go to: https://play.decentraland.zone/?renderer-branch=fix/non-supported-characters-are-displayed-incorrectly-for-notifications
2. Load into Decentraland with two avatars as a guests or via a wallets.
3. Write a chat message including non-supported special characters with one avatar.
4. Wait for the notification to appear for the second avatar.
5. Notice the non-supported characters are being replaced by `?`.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
